Update multiple data sources and add TSVB and Vega viz types (#7229)
opensearch-project/documentation-website: _dashboards/visualize/visbuilder.md

## Try VisBuilder in the OpenSearch Dashboards playground

> If you'd like to try out VisBuilder without installing OpenSearch locally, you can do so in the [OpenSearch Dashboards playground](https://playground.opensearch.org/app/vis-builder#/). VisBuilder is enabled by default.

Reviewer: ```suggestion
You can try VisBuilder without installing OpenSearch locally by using [OpenSearch Dashboards Playground](https://playground.opensearch.org/app/vis-builder#/). VisBuilder is enabled by default.
```
